app/models/ticket_desk.rb

```diff
@@ -1,4 +1,4 @@
 class TicketDesk < ApplicationRecord
-    has_and_belongs_to_many :cinema_halls
+    has_and_belongs_to_many :cinema_halls, :inverse_of=>:ticket_desk
```

Comment:
Keep consistency (see: https://github.com/KMosc/APITRY/pull/13/files#diff-103c7663225274279e2e159e0eec7a4fcaf32545109353486b5a95a18f3d3adaR12), please. It increases the readability of the code. Even better would be to start using a linter like [Rubocop](https://github.com/rubocop/rubocop)
```suggestion
    has_and_belongs_to_many :cinema_halls, :inverse_of=> :ticket_desk
```